.gitignore

```diff
@@ -4,3 +4,9 @@ node_modules
 .idea
 dev.config.json
 .DS_Store
+
+# price.log gererated by HelloCronScheduling
+price.log
+
+# vi temp files
+*.un~
```

Comment:
OPT: this is clearly very specific to your env, I would suggest you to put this in your personal global git ignore `core.excludeFiles` instead

Reply:
Thanks for that, didn't know that it's considered as a bad practice.